AbstractFactory.Andrew.University/AbstractFactory.Andrew.University/tests/UnitTest1.cs

```diff
@@ -0,0 +1,34 @@
+using System;
+using System.Collections.Generic;
+using Xunit;
+
+namespace AbstractFactory.Andrew.University.src
+{
+    public class UnitTest1
+    {
+        [Fact]
+        public void TestEngineeringCollege()
+        {
+            EngineeringFactory _engineering = new EngineeringFactory();
```

Comment:
style:  I'm not a big fan of being prescriptive about style, but underscore prefixes on local variables is unusual.  When I see underscore, I usually think about private fields (it's even a somewhat common convention in JS).  I don't personally prefix private fields with underscores anymore - and never for local variables.